processor/src/decorators/mod.rs

```diff
@@ -113,23 +113,23 @@ where
         let (q_hi, q_lo) = u64_to_u32_elements(quotient);
         let (r_hi, r_lo) = u64_to_u32_elements(remainder);
 
-        self.advice_provider.write_tape(AdviceSource::Value(r_hi))?;
-        self.advice_provider.write_tape(AdviceSource::Value(r_lo))?;
-        self.advice_provider.write_tape(AdviceSource::Value(q_hi))?;
-        self.advice_provider.write_tape(AdviceSource::Value(q_lo))?;
+        self.advice_provider.write_stack(AdviceSource::Value(r_hi))?;
+        self.advice_provider.write_stack(AdviceSource::Value(r_lo))?;
+        self.advice_provider.write_stack(AdviceSource::Value(q_hi))?;
+        self.advice_provider.write_stack(AdviceSource::Value(q_lo))?;
 
         Ok(())
     }
 
-    /// Injects a list of field elements at the front of the advice tape. The list is looked up in
+    /// Injects a list of field elements at the front of the advice stack. The list is looked up in
     /// the key-value map maintained by the advice provider using the top 4 elements on the stack
     /// as the key.
```

Comment:
Let's update this to: "Pushes a list of field elements onto the advice stack. The list is looked up in the advice map using the top 4 elements from the operand stack as the key."

Reply:
Done!